### ISSUE TITLE
RouteResolver argument counts not equals

### ISSUE THREAD

**Issue by riso348** [stein189/SimpleRouting#23]
`$router->get('/admin_crm/{w:controller}/{a:action}/{?:id}', function ($controller, $action, $id = null) {`

`Warning: array_combine(): Both parameters should have an equal number of elements in /var/www/..../web/vendor/szenis/routing/src/RouteResolver.php on line 44`

so array_combine return false. You do not control ?:{...} optional arguments ?
this worked in 1.x previous version.

try 

```
$count = min(count($route->getArguments()), count($arguments));
                $arguments = array_combine(array_slice($route->getArguments(), 0, $count), array_slice($arguments, 0, $count));
```
or compare optional arguments before please.

**Comment by stein189** [comment]
I will take a look into it

**Comment by stein189** [comment]
This issue is fixed in version 2.0.1
